src/main/java/emt/tile/generator/TileEntityBaseGenerator.java

```diff
@@ -336,14 +336,18 @@ public void inputintoGTnet() {
         if (!side)
             return;
 
+        boolean foundGregTechTile = false;
+
         for (byte i = 0; i < 6; i = (byte) (i + 1)) {
             if (getIGregTechTileEntityAtSide(i) != null) {
-                if (isUniversalEnergyStored(getOutputVoltage() * getOutputAmperage())) {
-                    long tEU = IEnergyConnected.Util.emitEnergyToNetwork(getOutputVoltage(), getOutputAmperage(), this);
-                    drainEnergyUnits(i, getOutputVoltage(), tEU);
-                }
+                foundGregTechTile = true;
```

Comment:
you can break the loop after you found a TE, maybe outsource this into a seperate method;
```JAVA
private boolean isAnyGregtechTileConnected() {
    for (int i = 0; i < 6; i++)
            if (getIGregTechTileEntityAtSide((byte) i) != null)
               return true;
    return false;
}
```

Reply:
Or if you want to make it funktional:
```JAVA
 private boolean isAnyGregtechTileConnected() {
  return IntStream.rangeClosed(0,5)
                .anyMatch(x ->
                        Objects.nonNull(getIGregTechTileEntityAtSide((byte)x))
                );
}